src/Components/WebAssembly/Server/src/ComponentsWebAssemblyApplicationBuilderExtensions.cs

```diff
@@ -76,6 +76,7 @@ private static StaticFileOptions CreateStaticFilesOptions(IFileProvider webRootF
             // release builds unless BlazorEnableDebugging is explicitly set to true.
             AddMapping(contentTypeProvider, ".pdb", MediaTypeNames.Application.Octet);
             AddMapping(contentTypeProvider, ".br", MediaTypeNames.Application.Octet);
+            AddMapping(contentTypeProvider, ".dat", MediaTypeNames.Application.Octet);
```

Comment:
FYI @javiercn. The timezone file that was added is named `dotnet.timezones.dat` which required adding this. I hadn't noticed this issue until I published. Also added the extension to the standalone.web.config.